### DISCUSSION TITLE
Add arrows to the shape tool

### DISCUSSION THREAD

**Discussion by thanighope** [PintaProject/Pinta#1175] [Ideas]
Could arrows/callouts be added to the shape tool?

![image](https://github.com/user-attachments/assets/40f500c6-9da2-4647-b875-09c249a61c61)

**Comment by thanighope** [comment]
this is where I would like to see arrows/callouts...
![image](https://github.com/user-attachments/assets/ce2dcf41-befe-46cf-a8e2-7510a73c55e6)

**Comment by cameronwhite** [comment]
Note you can make arrows using the Line / Curve tool , at either end of the curve

**Comment by bobinush** [comment]
I guess the version 2 of this feature request is to have some basic "custom shapes" similar to in the picture, in a "custom shape tool". I know I would love to have that.
